src/Main.java

```diff
@@ -0,0 +1,22 @@
+public class Main {
+    public static void main(String[] args) {
+        Sorting sort = new Sorting();
```

Comment:
I don't like that the class storing players is named Sorting. It is misleading and doesn't say that it stores players